### DISCUSSION TITLE
Can't run the backend because connection `:1.72` is not allowed

### DISCUSSION THREAD

**Discussion by Zelzahn** [ErikReider/SwayOSD#30] [Q&A] [answered]
When I try to run the backend I get the following error:
```
Error: org.freedesktop.DBus.Error.AccessDenied: Connection ":1.72" is not allowed to own the service "org.erikreider.swayosd" due to security policies in the configuration file
```
I honestly have no clue what the cause could be for this error and how I could fix this.

I followed the manual installation, Arch linux OS and Hyprland DE.

**Comment by ErikReider** [comment, marked as answer]
Running `meson setup` with `--prefix /usr` before building should fix your issue. Otherwise the policy files might not be in the correct location

**Reply by Zelzahn** [reply]
Thank you very much, this fixed the issue for me :partying_face:
